lib/gollum/app.rb

```diff
@@ -111,7 +116,37 @@ class App < Sinatra::Base
     end
 
     namespace '/gollum' do
-      
+      get '/feed/' do
+        url = env['REQUEST_URI'] || "http#{env['HTTPS'] == 'off' ? nil : 's'}://#{env['SERVER_NAME']}#{env['PATH_INFO']}" # Annoyingly, env['REQUEST_URI'] is not available in the tests. In case it sometimes proves to be unavailable in production, fall back to joining together the various parts of the url.
```

Comment:
I would leave out `Annoyingly`.

Reply:
Instead of `nil` I would prefer `''` since you're working with Strings.

Reply:
Perhaps `"#{env['HTTPS'] == 'off' ? 'http' : 'https'}` is more readable?

Reply:
yes, thanks!